src/main/java/uk/gov/companieshouse/company/profile/util/LinkRequest.java

```diff
@@ -8,9 +8,15 @@ public class LinkRequest {
     public static final String EXEMPTIONS_LINK_TYPE = "exemptions";
     public static final String EXEMPTIONS_DELTA_TYPE = "exemption_delta";
     public static final Function<Links, String> EXEMPTIONS_GET = Links::getExemptions;
+
     public static final String OFFICERS_LINK_TYPE = "officers";
     public static final String OFFICERS_DELTA_TYPE = "officer_delta";
     public static final Function<Links, String> OFFICERS_GET = Links::getOfficers;
+
+    public static final String PSC_LINK_TYPE = "psc";
```

Comment:
think it should be persons-with-significant-control instead of psc

Reply:
oh possibly.. i just took psc from this part of the ticket: "the location response header should be /company/{company_number}/links/psc"